aospy/test/test_calc_basic.py

```diff
@@ -275,5 +278,57 @@ def test_recursive_calculation(recursive_test_params):
     xr.testing.assert_equal(expected, result)
 
 
+def test_compute_pressure():
+    calc = Calc(
+        intvl_out='ann',
+        dtype_out_time='av',
+        var=p,
+        proj=example_proj,
+        model=example_model,
+        run=example_run,
+        date_range=('0006', '0006'),
+        intvl_in='monthly',
+        dtype_in_time='ts',
+        dtype_in_vert='sigma',
+        dtype_out_vert=None
+    )
+    calc.compute()
+    _test_files_and_attrs(calc, 'av')
+
+
+def test_compute_pressure_thicknesses():
+    calc = Calc(
+        intvl_out='ann',
+        dtype_out_time='av',
+        var=dp,
+        proj=example_proj,
+        model=example_model,
+        run=example_run,
+        date_range=('0006', '0006'),
+        intvl_in='monthly',
+        dtype_in_time='ts',
+        dtype_in_vert='sigma',
+        dtype_out_vert=None
+    )
+    calc.compute()
+    _test_files_and_attrs(calc, 'av')
+
+
+@pytest.mark.parametrize(
+    ['dtype_in_vert', 'expected'],
+    [(ETA_STR, [p_eta, dp_eta, condensation_rain, 5]),
+     ('pressure', [p_level, dp_level, condensation_rain, 5])])
+def test_replace_pressure(dtype_in_vert, expected):
+    arguments = [p, dp, condensation_rain, 5]
+    p_in, dp_in, _, _ = arguments
```

Comment:
Minor improvement: rather than discarding the non-pressure arguments, assert that they are identical before/after the call